config/jail.conf

```diff
@@ -767,3 +767,18 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow]
+# allow FTP traffic after successful HTTP auth
+enabled    = false
+filter     = apache-pass
+banaction  = iptables-multiport
+blocktype  = RETURN
+returntype = DROP
+# access log of the website with HTTP auth
+logpath    = /var/log/apache2/access.log
```

Comment:
Come on man...you edited your comment from access log to error log after I pointed out the discrepancy....